takwimu/templates/takwimu/_includes/profile/sections_list_detail.html

```diff
@@ -1,26 +1,28 @@
 {% if section.title|default:"TKW" == "TKW" and page.subpage_types|first == 'takwimu.ProfileSectionPage' %}
   <a>
-    <div class="jumbotron jumbotron-fluid border-top bg-light pt-4 pb-5 mb-0">
-      <div class="position-absolute" style="right: 12%; margin-top: -30px;">
-        <span class="bg-primary text-light py-2 px-3"><i class="fa fa-eye mr-1"></i> YOU ARE HERE</span>
+    <div class="jumbotron jumbotron-fluid border-top border-bottom bg-light pt-4 pb-5 mb-0">
+      <div class="position-absolute" style="right: 17%; margin-top: -30px;">
+        <span class="text-light py-3 px-4" style="background-color: #1e4a7e; box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.26);"><i class="fa fa-eye mr-1"></i> YOU ARE HERE</span>
       </div>
 {% elif section.title == page.title %}
   <a>
-    <div class="jumbotron jumbotron-fluid border-top bg-light pt-4 pb-5 mb-0">
-      <div class="position-absolute" style="right: 12%; margin-top: -30px;">
-        <span class="bg-primary text-light py-2 px-3"><i class="fa fa-eye mr-1"></i> YOU ARE HERE</span>
+    <div class="jumbotron jumbotron-fluid border-top border-bottom bg-light pt-4 pb-5 mb-0">
+      <div class="position-absolute" style="right: 17%; margin-top: -30px;">
+        <span class="text-light py-3 px-4" style="background-color: #1e4a7e; box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.26);"><i class="fa fa-eye mr-1"></i> YOU ARE HERE</span>
       </div>
 {% elif page.subpage_types|first == 'takwimu.ProfileSectionPage' %}
   <a href="{{ section.url|default:'#' }}">
-    <div class="jumbotron jumbotron-fluid border-top bg-white pt-4 pb-5 mb-0">
+    <div class=" container jumbotron jumbotron-fluid border-top bg-white pt-4 pb-5 mb-0">
 {% else %}
   <a href="{{ section.url|default:'../' }}">
-    <div class="jumbotron jumbotron-fluid border-top bg-white pt-4 pb-5 mb-0">
+      <div class="container jumbotron jumbotron-fluid border-top bg-white pt-4 pb-5 mb-0">
```

Comment:
Was just about to write about this `.container` class. Well done @KhadijaMahanga.